certbot/cert_manager.py

```diff
@@ -30,6 +33,29 @@ def update_live_symlinks(config):
             configuration.RenewerConfiguration(renewer_config),
             update_symlinks=True)
 
+def rename_lineage(config):
+    """Rename the specified lineage to the new name.
+
+    :param config: Configuration.
+    :type config: :class:`certbot.interfaces.IConfig`
+
+    """
+    if not config.certname:
+        raise errors.ConfigurationError("Specify a certificate name with "
+            "with flag --cert-name.")
+    if not config.new_certname:
+        raise errors.ConfigurationError("Specify a new name for certificate {0} "
```

Comment:
This should be `errors.MissingCommandlineFlag`.

But even better would be to do something like calling `disp.input("Enter the new name for certificate {0}".format(config.certname), flag="--new-cert-name")`. That will raise a nicely formatted `errors.MissingCommandlineFlag` for you.